scripts/load_data.sh

```diff
@@ -5,9 +5,16 @@ apk -q --no-cache add curl jq
 
 pip -q install s3cmd
 
-for file in NA12878.bam NA12878_20k_b37.bam; do
+FILES="NA12878.bam NA12878.bai NA12878_20k_b37.bam NA12878_20k_b37.bai"
+for file in ${FILES}; do
     curl -s -L -o $file "https://github.com/ga4gh/htsget-refserver/raw/main/data/gcp/gatk-test-data/wgs_bam/$file"
 
+    if [ "${file: -4}" = ".bai" ]; then
```

Comment:
It's more common to use this
```suggestion
    if [[ $file == *.bai ]]; then
```

Reply:
used the suggested `case` construct instead, for posix compliance